sql-plugin/src/main/java/com/nvidia/spark/rapids/ColumnarCopyHelper.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids;
+
+import ai.rapids.cudf.HostColumnVector.ColumnBuilder;
+
+import org.apache.spark.sql.vectorized.ColumnVector;
+
+/**
+ * A helper class which efficiently transfers different types of host columnar data into cuDF.
+ */
+public class ColumnarCopyHelper {
+
+  public static void nullCopy(ColumnBuilder b, int rows) {
+    for (int i = 0; i < rows; i++) {
+      b.appendNull();
+    }
+  }
+
+  public static void booleanCopy(ColumnVector cv, ColumnBuilder b, int rows) {
+    if (!cv.hasNull()) {
+      for (int i = 0; i < rows; i++) b.append(cv.getBoolean(i));
+      return;
+    }
+    for (int i = 0; i < rows; i++) {
+      if (cv.isNullAt(i)) {
+        b.appendNull();
+        continue;
```

Comment:
nit: Is there any performance difference between using the continue here and putting the `b.append(cv.getBoolean(i));` in an else? if not I would prefer the else.

Reply:
Done.